exchanges/kraken/kraken_test.go

```diff
@@ -99,6 +99,44 @@ func TestWrapperGetServerTime(t *testing.T) {
 	}
 }
 
+func TestUpdateOrderExecutionLimits(t *testing.T) {
+	t.Parallel()
+
+	type limitTest struct {
+		pair currency.Pair
+		step float64
+		min  float64
+	}
+
+	tests := map[asset.Item][]limitTest{
+		asset.Spot: {
+			{currency.NewPair(currency.ETH, currency.USDT), 0.01, 0.01},
+			{currency.NewPair(currency.XBT, currency.USDT), 0.1, 0.0001},
+		},
+	}
+
+	for assetItem, limitTests := range tests {
+		if err := k.UpdateOrderExecutionLimits(context.Background(), assetItem); err != nil {
+			t.Errorf("Error fetching %s pairs for test: %v", assetItem, err)
+		}
+
+		for _, limitTest := range limitTests {
```

Comment:
I think it would have been better for code readability's sake to name the `limitTest` var in this loop to something different to the inline struct you defined earlier to avoid collisions